trade-finance-manager-ui/server/nunjucks-configuration/filter-formatBooleanAsString.js

```diff
@@ -0,0 +1,13 @@
+const formatBooleanAsString = (bool) => {
```

Comment:
Maybe we could have `formatOptionalBooleanAsString`?

Reply:
Also sorry -- I know this is somewhat pointless, but could we have some tests for this please

Reply:
Felt bad so I thought I'd help you out with them :) 
```
const { formatBooleanAsString } = require('./filter-formatBooleanAsString');

describe('nunjuck filters - formatBooleanAsString', () => {
  const testCases = [
    { description: 'true', input: true, expectedOutput: 'Yes' },
    { description: 'false', input: false, expectedOutput: 'No' },
    { description: 'undefined', input: undefined, expectedOutput: '-' },
  ];

  it.each(testCases)(`should return $expectedOutput when input is $description`, ({ input, expectedOutput }) => {
    const result = formatBooleanAsString(input);
    expect(result).toEqual(expectedOutput);
  });
});

```

Reply:
Good idea, and yep totally mean to have tests! Now added, thanks for speeding that up! (and I like how you formatted that test!)